src/types/Todo.ts

```diff
@@ -0,0 +1,9 @@
+import { User } from './User';
+
+export interface Todo {
+  id: number,
+  userId: number,
+  title: string,
+  completed: boolean,
+  user?: User | null,
```

Comment:
I think you won't have cases when the user is undefined
```suggestion
  user: User | null,
```